src/Domain/DomainHelpers.cpp

```diff
@@ -309,6 +309,27 @@ void set_periodic_boundaries(
   }
 }
 
+std::array<OrientationMap<3>, 6> all_wedge_directions() {
```

Comment:
We decided (Gabe, Eamonn, Will, Larry) to have a helper function to create all six wedges at once that would inline these OrientationMaps